BricklinkSharp.Client/BricklinkClient.cs

```diff
@@ -840,12 +840,11 @@ public void Dispose()
     }
 
 
-    private async void _measureRequest(CancellationToken cancellationToken = default)
+    private async Task MeasureRequestAsync(CancellationToken cancellationToken = default)
     {
-        if (this._requesHandler != null)
+        if (this._requestHandler != null)
```

Comment:
One last issue (sorry), but the underscore for private fields is used to not require "this" ;)
```suggestion
        if (_requestHandler != null)
```